src/UI/AsyncControlTrait.php

```diff
@@ -43,9 +43,7 @@ public function handleAsyncLoad(): void
 	public function renderAsync(string $linkMessage = NULL, array $linkAttributes = NULL): void
 	{
 		$template = $this->createTemplate();
-		if ($template instanceof Template) {
-			$template->add('link', new AsyncControlLink($linkMessage, $linkAttributes));
-		}
+		$template->link =  new AsyncControlLink($linkMessage, $linkAttributes);
```

Comment:
Tady jsou dvě mezery za rovnítkem
```suggestion
		$template->link = new AsyncControlLink($linkMessage, $linkAttributes);
```

Reply:
Díky za postřeh. Upraveno